srv/modules/runners/sharedsecret.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+import os
+
+
+def show():
+    filename = '/etc/salt/master.d/sharedsecret.conf'
```

Comment:
That I cannot do. This is an "internal" file expected by the sharedsecret eauth module.